migrations/V98__Convert_LCID_Extension_Columns_To_NULL.sql

```diff
@@ -0,0 +1,8 @@
+-- Some actions were created in some deployed environments with "" as the value for these columns instead of NULL.
+
+ALTER TABLE actions ADD COLUMN lcid_expiration_change_new_scope text;
```

Comment:
This file looks like the same as the V97 migration that added these columns originally; is this supposed to have an `UPDATE` statement for these tables that converts `""` to `NULL`?

Reply:
Yes this somehow got messed up in his commit, he had a version with an update call on all columns with "" to NULL

Reply:
Yeah - I copied the migration and looks like I messed it up when I committed, lol. 😓

Reply:
Resolved in https://github.com/CMSgov/easi-app/pull/1440/commits/945d7327e212c37bb65ea65284e5a8649ff13323.